sdk/kusto/Azure.ResourceManager.Kusto/samples/Generated/Samples/Sample_KustoDatabaseResource.cs

```diff
@@ -369,10 +369,10 @@ public async Task CheckKustoScriptNameAvailability_KustoScriptsCheckNameAvailabi
 
         [Test]
         [Ignore("Only validating compilation of examples")]
-        public async Task ValidateDataConnection_KustoDataConnectionEventGridValidation()
+        public async Task DataConnectionValidationDataConnection_KustoDataConnectionEventGridValidation()
         {
             // Generated from example definition: specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2023-08-15/examples/KustoDataConnectionEventGridValidationAsync.json
-            // this example is just showing the usage of "DataConnections_DataConnectionValidation" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "DataConnections_dataConnectionValidation" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `data`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2024-04-13/kusto.json#L3661